### ISSUE TITLE
Svg xlink error

### ISSUE THREAD

**Issue by larsenwork** [pugjs/pug-lint#61]
When I'm adding svg files the linter gives me an error saying "No namespacing of attributes attribute allowed" 

```
svg
    use(xlink:href='#icon-pencil')
```

**Comment by benedfit** [comment]
HI @larsenwork 

Are you able to provide the config you are using, along with the version of `pug-lint` and `jade` that is causing this error as I'm currently unable to replicate this issue?

**Comment by larsenwork** [comment]
Hi @benedfit, I'm using the atom linter by @zyklus so he can probably give you some more detail https://github.com/d9d/atom-linter-jade/issues/9#event-551190294

**Comment by benedfit** [comment]
@larsenwork unfortunately that particular issue isn't being raised to `pug-lint` but in fact, `jadelint`, a no longer maintained linter that the Atom linter also used along side `pug-lint`.

I've added an additional comment to your original issue to see if @zyklus has any plans to update the atom linter

**Comment by benedfit** [comment]
Feature request for atom linter to drop support for `jadelint`: https://github.com/d9d/atom-linter-jade/issues/11

**Comment by benedfit** [comment]
@larsenwork: May I suggest trying https://atom.io/packages/linter-pug instead. This linter only uses `pug-lint` so the error you were seeing should be resolved

**Comment by larsenwork** [comment]
👍 thanks
